### PR TITLE
Added sdk docs.

### PR DESCRIPTION
**Pull request by andyi2it**
Updated references in docs.
Updated sdk docs with api group and kserve name

Added an empty md file temporarily.

### CODE REVIEW

**Comment by andyi2it** [comment]
The v1alpha* references will be removed once they are deprecated.

**Comment by yuzisun** [comment]
thanks @Suresh-Nakkeran ! Any thoughts on how we can keep the sdk docs in sync as currently they are generated from kserve repo.

**Comment by andyi2it** [comment]
@yuzisun @Suresh-Nakkeran  explored about it for API reference. Once that is in place, we can automate that.
I was thinking of a workflow in kserve where if there is a potential doc change then it can checkout website repo and auto push changes to it and generate PR automatically.
@yuzisun  Whats your thoughts on it?

**Comment by yuzisun** [comment]
> The v1alpha* references will be removed once they are deprecated.

I think in docs we can update to v1beta1 now  and all other examples use v1beta1 API shape

File: docs/sdk_docs/docs/KServeClient.md [hunk 1]
```diff
@@ -0,0 +1,425 @@
...
+------------ | ------------- | ------------- | -------------
+inferenceservice  | [V1alpha2InferenceService](V1alpha2InferenceService.md) | InferenceService defination| Required |
+namespace | str | The InferenceService's namespace. If the `namespace` is not defined, will align with InferenceService definition, or use current or default namespace if namespace is not specified in InferenceService definition. | Optional|
+watch | bool | Watch the patched InferenceService if `True`, otherwise will return the replaced InferenceService object. Stop watching if InferenceService reaches the optional specified `timeout_seconds` or once the InferenceService overall status `READY` is `True`. | Optional |
+timeout_seconds | int | Timeout seconds for watching. Defaults to 600. | Optional |
+
+### Return type
+object
+
+## rollout_canary
```
Comment (yuzisun) [inline docs/sdk_docs/docs/KServeClient.md:283]: `rollout_canary` and `promote` are deprecated, can be removed from here

File: docs/sdk_docs/docs/KServeClient.md [hunk 2]
```diff
@@ -0,0 +1,425 @@
...
+
+Class | Method |  Description
+------------ | ------------- | -------------
+KServeClient | [set_credentials](#set_credentials) | Set Credentials|
+KServeClient | [create](#create) | Create InferenceService|
+KServeClient | [get](#get)    | Get or watch the specified InferenceService or all InferenceServices in the namespace |
+KServeClient | [patch](#patch)  | Patch the specified InferenceService|
+KServeClient | [replace](#replace) | Replace the specified InferenceService|
+KServeClient | [rollout_canary](#rollout_canary) | Rollout the traffic on `canary` version for specified InferenceService|
+KServeClient | [promote](#promote) | Promote the `canary` version of the InferenceService to `default`|
```
Comment (yuzisun) [inline docs/sdk_docs/docs/KServeClient.md:25]: These two APIs are deprecated, let's remove them.

File: docs/sdk_docs/docs/KServeClient.md [hunk 3]
```diff
@@ -0,0 +1,353 @@
...
+from kserve import V1beta1PredictorSpec
+from kserve import V1alpha2TensorflowSpec
+from kserve import V1beta1InferenceServiceSpec
+from kserve import V1beta1InferenceService
+from kserve import KServeClient
+
+default_model_spec = V1beta1EndpointSpec(predictor=V1beta1PredictorSpec(tensorflow=V1alpha2TensorflowSpec(
+    storage_uri='gs://kfserving-samples/models/tensorflow/flowers')))
+canary_model_spec = V1beta1EndpointSpec(predictor=V1beta1PredictorSpec(tensorflow=V1alpha2TensorflowSpec(
+    storage_uri='gs://kfserving-samples/models/tensorflow/flowers')))
```
Comment (yuzisun) [inline docs/sdk_docs/docs/KServeClient.md:202]: the canary API is changed in v1beta1, please help checkout e2e test example https://github.com/kserve/kserve/blob/master/test/e2e/predictor/test_canary.py#L32

File: docs/sdk_docs/docs/KServeClient.md [hunk 4]
```diff
@@ -0,0 +1,353 @@
...
+from kubernetes import client
+from kserve import constants
+from kserve import V1beta1EndpointSpec
+from kserve import V1beta1PredictorSpec
+from kserve import V1alpha2TensorflowSpec
+from kserve import V1beta1InferenceServiceSpec
+from kserve import V1beta1InferenceService
+from kserve import KServeClient
+
+default_endpoint_spec = V1beta1EndpointSpec(
```
Comment (yuzisun) [inline docs/sdk_docs/docs/KServeClient.md:246]: `EndpointSpec` does not exist anymore in v1beta1

**Review by yuzisun — APPROVED** [review]

**Pull request merged**